chaco/tools/line_inspector.py

```diff
@@ -78,7 +78,8 @@ def draw(self, gc, view_bounds=None):
         if self.is_listener:
             tmp = self._get_screen_pts()
         elif self.is_interactive:
-            tmp = self._last_position
```

Comment:
It looks like this (and some of the other changes in this module) changes the functionality. Can you explain why this is necessary? If this fixes a bug, it would probably be good to split it off from the current PR and submit as a separate PR.